onnxruntime/core/providers/cpu/cpu_execution_provider.cc

```diff
@@ -590,895 +592,897 @@ KernelCreateInfo BuildKernelCreateInfo<void>() {
 
 Status RegisterOnnxOperatorKernels(KernelRegistry& kernel_registry) {
   static const BuildKernelCreateInfoFn function_table[] = {
-    BuildKernelCreateInfo<void>,  //default entry to avoid the list become empty after ops-reducing
```

Comment:
Can the diffs in this file be made relevant to this PR?

Reply:
Diffs seem off. The previous indent of 2 looks more correct than the indent of 4 in the PR. 

---
In reply to: [518505101](https://github.com/microsoft/onnxruntime/pull/5665#discussion_r518505101) [](ancestors = 518505101)

Reply:
Sure- I don't know why it shows up like this - I ll fix it

Reply:
Fixed